monai/utils/type_conversion.py

```diff
@@ -83,7 +83,7 @@ def get_dtype(data: Any):
     return type(data)
 
 
-def convert_to_tensor(data, wrap_sequence: bool = False):
+def convert_to_tensor(data, wrap_sequence: bool = False, device: Optional[torch.device] = None):
```

Comment:
Hi @wyli ,

I am not sure we only support `device` arg or `kwargs` for `torch.to()` or `torch.as_tensor()` here?
What do you think?

Thanks.

Reply:
Right, 'to' supports some detailed options, 'as_tensor' doesn't support many, perhaps we can enhance it further in another PR if needed.

Reply:
Don't understand this integration error, I'll rerun the test https://github.com/Project-MONAI/MONAI/runs/3569135774